### ISSUE TITLE
Upload sourcemaps to Azure from CI

### ISSUE THREAD

**Issue by pl4nty** [pl4nty/anutimetable#91]
Allows unminifying code in Application Insights, [as documented](https://docs.microsoft.com/en-us/azure/azure-monitor/app/source-map-support). Would make troubleshooting from logs much easier. Not sure if this is possible given that Azure (Oryx) handles the build, but worth investigating.

**Comment by ashleylamont** [comment]
I would presume Oryx would pick up that it's a Webpack/CRA installation from Node deps or that there's a Node build script, and run it that way, in which case it would all go through webpack (which is admittedly obscured by CRA) which should be generating sourcemaps already I thought, unless it was disabled, in any case it shouldn't be terribly difficult to re-enable them I think? I'll have to look into it a little

**Comment by pl4nty** [comment]
@ashleylamont yeah it generates sourcemaps, but I'm not sure how to get those from Oryx into a Blob container

**Comment by pl4nty** [comment]
After digging a bit deeper, we would need to move builds from Oryx to Actions as discussed at Azure/static-web-apps#17. I'm ok with that, would just have to be more careful when applying the 'safe to test' label - PRs could potentially access secrets during the build process.
